packages/formstr-app/src/containers/CreateFormNew/components/AIFormGeneratorModal/ModelSelector.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Why are there two model selector compoonents? where is the other one used? in `src/components/ModelSelector` ?